magento/magento2
# Move menu entry log from level INFO to DEBUG

### Description (*)
The `var/log/system.log` file gets flooded with messages like the following.
```
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Theme::design_config was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Theme::system_design_theme was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Config::system_config was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::system_design_schedule was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::system_store was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::dashboard was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::system was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::system_tools was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::system_design was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::system_convert was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::system_cache was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::marketing was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::marketing_communications was processed [] []
[2021-09-03 11:44:16] main.INFO: Add of item with id Magento_Backend::marketing_seo was processed [] []
```

While these messages may be interesting to see in the `var/log/debug.log` file, it makes no sense for them to appear in `var/log/system.log`.

### Related Pull Requests
N/A

### Fixed Issues (if relevant)
None that I know of.

### Manual testing scenarios (*)
1. Flush cache.
2. Log in to backend.
3. See `var/log/debug.log`. The "Add of item ..."  messages should be seen here.
4. See `var/log/system.log`. The messages should not be seen here.

### Questions or comments
None.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38095: Move menu entry log from level INFO to DEBUG
